test/cloudinary_spec.js

```diff
@@ -775,6 +775,23 @@ describe("cloudinary", function () {
       responsive: true,
     });
   });
+  it("should generate urls with a 32 character signature when both sign_url and long_url_signature are true", function () {
+    var options, result;
+    options = {
+      sign_url: true,
+      long_url_signature: true,
+    };
+    result = cloudinary.utils.url("sample.jpg", options);
+    expect(result).to.eql('http://res.cloudinary.com/test123/image/upload/s--2hbrSMPOjj5BJ4xV7SgFbRDevFaQNUFf--/sample.jpg');
+  });
+  it("should generate urls with a 8 character signature when sign_url is true", function () {
+    var options, result;
+    options = {
+      sign_url: true,
+    };
+    result = cloudinary.utils.url("sample.jpg", options);
+    expect(result).to.eql('http://res.cloudinary.com/test123/image/upload/s--v2fTPYTu--/sample.jpg');
```

Comment:
can we generate this signature externally in the test, instead of hardcoding the result?
I think it will make it a bit easier to know what's being tested.
For example, it's hard to tell at a glance if s--v2fTPYTu-- is even a valid signature.

Reply:
The spec for this task defines that using certain inputs we can always expects a deterministic outcome producing a certain hardcoded string.

<img width="795" alt="CleanShot 2020-04-14 at 15 25 27@2x" src="https://user-images.githubusercontent.com/1785648/79224705-3ccb3080-7e64-11ea-8df2-d7963732dcbc.png">

This follows the guidelines for testing across SDKs we got from @const-cloudinary and Asi.

And it makes sense, as generating the signature in the test using the same method that the test is testing defeats the purpose of the test.

In pseudo code this would be akin to
```javascript
expect(sign(str)).to.eql(sign(str))
```